go/libkb/chain_link_v2_test.go

```diff
@@ -30,29 +40,31 @@ type outerLinkV2WithMetadataEmbedder struct {
 func TestOuterLinkV2WithMetadataEmbedderEncode(t *testing.T) {
 	var o outerLinkV2WithMetadataEmbedder
 	_, err := MsgpackEncode(o)
-	require.Equal(t, errCodecEncodeSelf, err)
+	requireErrorHasSuffix(t, errCodecEncodeSelf, err)
 }
 
 func TestOuterLinkV2WithMetadataEmbedderDecode(t *testing.T) {
 	var o outerLinkV2WithMetadataEmbedder
 	err := MsgpackDecode(&o, []byte{0x1, 0x2})
-	require.Equal(t, errCodecDecodeSelf, err)
+	requireErrorHasSuffix(t, errCodecDecodeSelf, err)
 }
 
 type outerLinkV2WithMetadataPointerEmbedder struct {
 	*OuterLinkV2WithMetadata
 }
 
 func TestOuterLinkV2WithMetadataPointerEmbedderEncode(t *testing.T) {
-	var o outerLinkV2WithMetadataPointerEmbedder
+	o := outerLinkV2WithMetadataPointerEmbedder{
```

Comment:
old codec still threw an error, but new codec skips encoding if it's nill I guess